json/src/machine.nr

```diff
@@ -278,8 +282,8 @@ pub fn rewrite_stack(input: RewriteStackInput) -> RewriteStackOutput {
     let to_clear_first = !not_to_clear_first;
 
     let tree_hash_change_value = [
-        Field::from(!to_clear_zeroth as u64) * next_state_hash_0,
-        Field::from(!to_clear_first as u64) * next_state_hash_1,
```

Comment:
This is printed as `0x6effffffffffffff91`, so I fixed to `(!to_clear_zeroth) as u64`.